### ISSUE TITLE
Terminal opened using shortcut is stuck at the top and It's cannot be moved into next window

### ISSUE THREAD

**Issue by dileep9490** [warpdotdev/Warp#3484]
### Describe the bug

I've assigned a shortcut (cmd + option + "/") to open warp and whenever i open it it's stuck at the top center and when i tired to move it to next screen it cannot be moved.but the normal window opened from the launch pad is working fine and can be moved
device: Macbook air M2 
version: ventura 13.5

i've tried to uninstall it and install it again but it's not working

### To reproduce

I don't know how to reproduce i just moved the warp terminal which is opened using shortcut and it got stuck after relaunching it later at the last position i.e, at the top center

### Operating system

MacOS

### Operating system and version

Ventura 13.5

### Shell Version

bash version 3.2.57(1)-release (arm64-apple-darwin22)

### Current Warp version

v0.2023.08.02.08.05.stable_03

### Regression

Yes, this bug started recently or with an X Warp version

### Does this block you from using Warp daily?

No

### Is this a Warp specific issue? (i.e. does it happen in Terminal, iTerm, Kitty, etc.)

Yes, this I confirmed this only happens in Warp, not other terminals.

### Warp Internal (ignore): linear-label:b8107fdf-ba31-488d-b103-d271c89cac3e

None

**Comment by dannyneira** [comment]
Hi @dileep9490, if you've assigned the shortcut in the Warp Settings > Features > Global Hotkey - Dedicated window, you can also set the position/location/screen that the popup window will render. If you set it to Active, you should also be able to resize the dedicated window and should retain the new size. 

![CleanShot 2023-08-07 at 14 39 27](https://github.com/warpdotdev/Warp/assets/16809145/bd5007a8-eadb-466d-930b-67a388d586d5)

Hope this helps with your issue. Please note that Warp must be running, at least in the background, for this feature to work. See more about this feature in our docs. https://docs.warp.dev/features/windows/hotkey-window

**Comment by dannyneira** [comment]
@dileep9490 any luck with the suggestions above?

**Comment by dileep9490** [comment]
@dannyneira yeah it's working now but would like it to move instead of it sticking at the top

**Comment by dannyneira** [comment]
Heard, it's a bit of a workaround, but you can adjust the borders of the Dedicated hotkey window to "move" it around, it can't be moved like a traditional window at this time. This along with the "Active" window setting should help some with your use case. 

https://github.com/warpdotdev/Warp/assets/16809145/b0acaacb-5748-427a-91a1-3ce4ea34aa21

**Comment by dileep9490** [comment]
This looks a long way to move it'll be easier and better when user can move it around than resizing it to change it's position. 
For example:
when i'm watching a youtube tutorial if it's on top it'll be blocking my view and i'll move it down let's just say then it'll be blocking when i'm seeing stuff from bottom when i'm reading an article.
what i want to say is most of the time we don't want terminal to be in a fixed position as it might come in between the flow when working
